src/test/java/se/sundsvall/messaging/integration/db/DbIntegrationTests.java

```diff
@@ -118,6 +123,33 @@ void test_saveHistory() {
         verify(mockHistoryRepository, times(1)).save(any(HistoryEntity.class));
     }
 
+    @Test
+    void test_getStats() {
```

Comment:
The "test_"-prefix is redundant since the method is annotaded `@Test` and exists in a class that ends with the suffix "Test".

Furthermore: it does not comply to the Java naming standards.


(comment is applicable on all places where the prefix "test_" exists)